addons/stock/models/stock_orderpoint.py

```diff
@@ -212,7 +214,28 @@ def action_stock_replenishment_info(self):
         action['res_id'] = res.id
         return action
 
+    @api.depends('product_id', 'qty_to_order', 'product_max_qty')
+    def _compute_unwanted_replenish(self):
+        for orderpoint in self:
+            orderpoint.unwanted_replenish = (orderpoint.product_id.virtual_available + orderpoint.qty_to_order) > orderpoint.product_max_qty
```

Comment:
For the virutal available. You should use the company of the orderpoint in the context

Reply:
float_compare instead of >